native/chat/chat-thread-list.react.js

```diff
@@ -47,6 +48,7 @@ type Props = {|
   navigation: ChatNavigationProp<'ChatThreadList'>,
```

Comment:
This prop is no longer typed correctly. `ChatThreadList` is now a screen of your new navigator. You'll need to define a new `ChatTopTabsNavigationProp`, similar to how `ChatNavigationProp` is defined

Reply:
Please take a look at new commit, I'm not exactly sure if I did this correctly